drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/AdditionalInfoPage.java

```diff
@@ -135,10 +151,29 @@ void setupUpdateEngineWithChanges() {
         }
     }
 
-    private boolean isHeaderCompleted() {
+    private boolean isHeaderNotEmpty() {
         return headerEnabled && !nil(getHeader());
     }
 
+    private boolean isHeaderUnique() {
+        final String header = getHeader();
+        for (final ActionCol52 col52 : presenter.getModel().getActionCols()) {
+            if (col52.getHeader().equals(header)) {
+                return false;
+            }
+        }
+
+        for (CompositeColumn<?> cc : presenter.getModel().getConditions()) {
+            for (int iChild = 0; iChild < cc.getChildColumns().size(); iChild++) {
+                if (cc.getChildColumns().get(iChild).getHeader().equals(header)) {
+                    return false;
+                }
+            }
+        }
```

Comment:
@jomarko I'm not sure about the consequences of changing this behavior. Why shouldn't we keep that? :-)